modular_stonehedge/licensed-death-rattler/Death-Rattler/code/Death-Rattler.dm

```diff
@@ -0,0 +1,158 @@
+// This code is part of Death-Rattler
+// Copyright (C) 2024 Moribund/Luctus-Comptus
+// Unauthorized use, modification, or distribution of this code is prohibited without copyright holders approval. See LICENSE.md
+
+// I hate doing this. basically if you are anyone but ratwood/gyran DM me first and you can use whatever the fuck your heart wants from this. not only did he brag about theft and how easy it is he tryed to pull this same shit.
+
+
+
+
+/obj/item/storage/fancy/shhig
+	name = "Shhig Brand Zigs"
```

Comment:
```suggestion
	name = "\improper Shhig Brand Zigs"
	gender = PLURAL
```
this will result in "some Shhig Brand Zigs" and "the Shhig Brand Zigs" when you use it in an embedded expression, like so
![image](https://github.com/user-attachments/assets/a24a12f1-9fe3-4ee4-b49b-bfaf4f625a1a)